assets/js/modules/analytics-4/index.js

```diff
@@ -701,6 +703,18 @@ export const registerWidgets = ( widgets ) => {
 		},
 		[ AREA_MAIN_DASHBOARD_KEY_METRICS_PRIMARY ]
 	);
+
+	widgets.registerWidget(
+		'keyMetricsEventDetectionCalloutNotification',
+		{
+			Component: ConversionReportingNotificationCTAWidget,
+			width: [ widgets.WIDGET_WIDTHS.FULL ],
+			priority: 0,
+			modules: [ 'analytics-4' ],
+			isActive: () => isFeatureEnabled( 'conversionReporting' ),
```

Comment:
Note to reviewers: `isActive` runs before every other check except module area so it should be used as conservatively as possible with simple checks only.